lib/ide/ast/impl/src/lib.rs

```diff
@@ -152,6 +152,28 @@ impl Ast {
         self
     }
 
+    // TODO use strongly typed IDMap, after it's merged to master
```

Comment:
Not a request (as the dependency is currently extremely small and we will soon merge it anyway) but you can also create PRs against branches, so that way you can develop features depending on not yet merged features.
(though merges can get really painful because we are squashing the PRs into single commit)